wg-multitenancy/2021-annual-report.md

```diff
@@ -0,0 +1,51 @@
+# 2021 WG Multitenancy Annual Report
+
+### What was the initial mission of the group and if it's changed, how?
+
+**Initial Mission:**
+
+Define the models of multitenancy that Kubernetes will support. Discuss and execute upon any
+remaining work that needs to be done to support these models. Create conformance tests that 
+will prove that these models can be built and used in production environments.
+
+**Current Mission:**
+
+We are focusing more on projects using Kubernetes than trying to directly change the API machinery, 
+which is a huge and potentially intractable problem. We are continuing to work on conformance 
+testing as part of one of our projects.
+
+### What’s the current roadmap until completion?
+
+**We have three projects we are incubating:**
```

Comment:
This is great! Is there also a rough definition of what completion might look like?

Reply:
That's a hard question. Is it when true multi-tenancy capabilities are added to the control plane? Or is it when we don't have projects incubating and let the longer term concerns fall to a SIG, using the large amount of documentation and mindshare the WG has built up over the past few years? I'm open to discussion on which benefits k8s most.

Reply:
My take on this is having a sense for:

1. do the current projects have a destination they're tracking toward (e.g. once the benchmarks listed as in-progress are complete, will that project be considered finished?)

2. once the listed projects are finished, does the working group plan to start new projects or consider its mission complete?

Reply:
Apologies, should have phrased the question better. What @liggitt said. :)

Reply:
Gotcha, yeah. I documented our project plan here: https://docs.google.com/document/d/1U8RQQmTUjxgMZY05HG2f7b3KsB94BhK4Ko6aWbLNXcc/edit

Reply:
thanks, added that link in. will let @nikhita take a look and resolve the comment thread.